src/main/scala/zooklabs/program/OverallScoreCalculations.scala

```diff
@@ -0,0 +1,38 @@
+package zooklabs.program
+
+import zooklabs.repository.model.{LeagueCounts, LeagueRanks}
+
+object OverallScoreCalculations {
+  val MaximumScore = 50_000L
+
+  val square: Double => Double = (x: Double) => x * x
+
+  // We want biggest to be best
+  val reverseScore: Long => Long = (score:Long) => MaximumScore - score
+
+  val processNormalised: Double => Long = square andThen Math.round
+
+  // Converts a position on the scale 1 to (entries) to 0 to 100
+  def normaliseRank(position: Int, entries: Int): Double = {
+    (position - 1) * (100.0 / entries)
+  }
+
+  def getSingleLeagueScore(position: Int, entries: Int): Long = {
+    val normalised = normaliseRank(position, entries)
+
+    processNormalised( normalised )
```

Comment:
```suggestion
    normaliseRank(position, entries) andThen processNormalised( normalised )
```

Reply:
Can't compile that but can do something like

`val normalise: Int => Double = normaliseRank(_, entries) 
(normalise andThen processNormalised)(position)`

And maybe curry normaliseRank to make it a bit more gracious